language-design.md

```diff
@@ -106,7 +254,19 @@ you do so? If not, why not?
 
 **Response**
 
+We believe that these two experiences of natural languages are not at odds with one another. Pavlus' statement that natural language replacements are more readable is correct. A simple example is the readability of for loops in Python and C++. When introducing for-loops to new programmers, the Python language is much more intuitive in expressing the actions of the program. We do recognize that _natural_ language as a programming language is extraordinarily difficult since natural language relies both on the context of speaker and the context of the listener. Trying to fit a natural language into the confines of a strict interpretation is the same issue that automated translators have. (As a fun aside, try to translate some phrases in Google Translate and then translate them back into English). Even the interpretation of simple phrases is not one-to-one and is often many-to-many, unideal for programming languages and their interpreters. We posit that natural language in a programming languages _is_ beneficial, but to an extent. We are unsure where the boundary lies between too much and too little but we will discuss this later. Historically, we find that programming languages generally neglect the clarity that natural language can provide.
+
+In the design of a DSL, we would both elect to include natural language, to an extent. We could select a word that expresses an idea but avoid phrases or words that might express multiple ideas, at least to an audience similar to ourselves. We recognize that different contexts could make the natural language bits strange and unintuitive, but we believe the best way to avoid that is to stick to smaller phrases or words. (In our discussion, we chose to call these small phrases or words language elements.) Natural language is a tool avaiable to us to make a DSL more intuitive though a tool that we must be especially careful not to misuse. The danger of context driving a different interpreters of code is always present.
```

Comment:
Does this only correspond to using small natural language fragments as symbols, like in Python, or are there situations where you think language that required more sophisticated parsing would be a good idea? (Personally, I think that at least for "languages as we know them", it would not be a good idea.)